src/Microsoft.OpenApi.OData.Reader/Common/EdmModelHelper.cs

```diff
@@ -309,24 +310,24 @@ internal static string GenerateODataTypeCastPathOperationIdPrefix(ODataPath path
             string operationId = null;
             if (secondLastSegment is ODataComplexPropertySegment complexSegment)
             {
-                string listOrGet = complexSegment.Property.Type.IsCollection() ? "List" : "Get";
+                string listOrGet = includeListOrGetPrefix ? (complexSegment.Property.Type.IsCollection() ? "List" : "Get") : null;
```

Comment:
Less work when we enable NRT this way
```suggestion
                string listOrGet = includeListOrGetPrefix ? (complexSegment.Property.Type.IsCollection() ? "List" : "Get") : string.Empty;
```